masterlist.yaml

```
@@ -17651,6 +17656,11 @@ plugins:
       - <<: *patchProvided
         subs: [ 'Immersive Citizens - AI Overhaul' ]
         condition: 'active("Immersive Citizens - AI Overhaul.esp") and not active("CFTO - Immersive Citizens Patch.esp")'
+      - <<: *patch3rdParty
+        subs:
+          - 'Creation Club Bittercup'
+          - '[CFTO Bittercup Compatibility Patch](https://www.nexusmods.com/skyrimspecialedition/mods/60587/)' 
+        conditions: 'active("CFTO.esp") and active("cckrtsse001_altar.esl") and not active("CFTO_Bittercup_Patch.esp")'    
```

1. As stated above, the active check isn't necessary here.
2. There are some extraneous whitespaces at the ends of a few lines.
3. This message should be sorted alphanumerically by alias name (i.e., `patch3rdParty` goes above `patchProvided` & below `alsoUseX`). You'll have to do this manually; I can't make a suggestion for it.

```suggestion
      - <<: *patch3rdParty
        subs:
          - 'Creation Club Bittercup'
          - '[CFTO Bittercup Compatibility Patch](https://www.nexusmods.com/skyrimspecialedition/mods/60587/)'
        conditions: 'active("cckrtsse001_altar.esl") and not active("CFTO_Bittercup_Patch.esp")'
```